.prototools

```diff
@@ -1 +1,2 @@
 bun = "1.1.20"
+node = "22.9.0"
```

Comment:
bun 使ってるから node いらないと思ってたけど、なぜ node？

Reply:
Nodeのバージョンによって`astro sync`が失敗したり成功したりする
これのせいで、Nodeのバージョンが20+じゃないと`astro dev`してもContent Collectionsの型が生成されなくて`any`になる

```
 ReoHakase in  shikosai33-web on   2/add-content-sc…  📦1  📝2 
❯ node -v
v22.9.0

 ReoHakase in  shikosai33-web on   2/add-content-sc…  📦1  📝2 
❯ bun astro sync
(node:68665) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
13:33:05 [types] Added src/env.d.ts type declarations.
13:33:05 [types] Added src/env.d.ts type declarations.
13:33:05 Types generated 497ms

 ReoHakase in  shikosai33-web on   2/add-content-sc…  📦1  📝2 
❯ proto i node 18 --pin local
Node.js 18.19.1 has already been installed at ~/.proto/tools/node/18.19.1

 ReoHakase in  shikosai33-web on   2/add-content-sc…  📦1  📝3 
❯ node -v
v18.19.1

 ReoHakase in  shikosai33-web on   2/add-content-sc…  📦1  📝3 
❯ bun astro sync
SyntaxError: Unexpected token 'with'
    at ModuleLoader.moduleStrategy (node:internal/modules/esm/translators:152:18)
    at ModuleLoader.moduleProvider (node:internal/modules/esm/loader:298:14)
error: "astro" exited with code 1

 ReoHakase in  shikosai33-web on   2/add-content-sc…  📦1  📝3 
```

Reply:
`--bun`オプションつけたら動いたかも

```
 ReoHakase in  shikosai33-web on   2/add-content-sc…  📦1  📝3 
❯ proto i node 18 --pin local
Node.js 18.19.1 has already been installed at ~/.proto/tools/node/18.19.1

 ReoHakase in  shikosai33-web on   2/add-content-sc…  📦1  📝3 
❯ bun --bun astro sync
13:36:09 [types] Added src/env.d.ts type declarations.
13:36:09 [vite] Re-optimizing dependencies because lockfile has changed
13:36:09 [types] Added src/env.d.ts type declarations.
13:36:09 Types generated 345ms

 ReoHakase in  shikosai33-web on   2/add-content-sc…  📦1  📝3 
❯ bun astro sync
SyntaxError: Unexpected token 'with'
    at ModuleLoader.moduleStrategy (node:internal/modules/esm/translators:152:18)
    at ModuleLoader.moduleProvider (node:internal/modules/esm/loader:298:14)
error: "astro" exited with code 1
```

Reply:
それで動けば無問題

Reply:
問題なさそうだったから消した、確認よろ